nx/scripts/postlcp.js

```diff
@@ -1,6 +1,10 @@
 import { loadBlock } from './nexter.js';
+import { sampleRUM } from '../deps/rum.js';
 
 (async function loadPostLCP() {
+  window.hlx = window.hlx || {};
```

Comment:
These should be moved to `rum.js` as they are part of the rum feature. We want to keep these things encapsulated to their own area of concern.